### PR TITLE
New version: Google.Chrome.Dev version 99.0.4818.2

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/40521)

### CODE REVIEW

File: manifests/g/Google/Chrome/Dev/99.0.4818.2/Google.Chrome.Dev.installer.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
-# Created with YamlCreate.ps1 v2.0.4 $debug=AUSU.7-2-0
+# Created with YamlCreate.ps1 v2.0.4 $debug=AUSU.7-2-1
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
 
 PackageIdentifier: Google.Chrome.Dev
-PackageVersion: 98.0.4758.9
+PackageVersion: 99.0.4818.2
```
Comment (OfficialEsco) [inline manifests/g/Google/Chrome/Dev/99.0.4818.2/Google.Chrome.Dev.installer.yaml:5]:
```suggestion
PackageVersion: 98.0.4758.11
```

**Review by OfficialEsco — COMMENTED** [review]
I'm seeing this version

File: manifests/g/Google/Chrome/Dev/99.0.4818.2/Google.Chrome.Dev.installer.yaml (hunk 1 above)
Comment (OfficialEsco) [reply manifests/g/Google/Chrome/Dev/99.0.4818.2/Google.Chrome.Dev.installer.yaml:5]: Aaaaand ATM i'm getting `98.0.4758.9`

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
I'm seeing `98.0.4758.9` instead of `99.0.4818.2`:
image

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**